src/GitHub.VisualStudio/Settings/PackageSettingsGen.cs

```diff
@@ -0,0 +1,34 @@
+﻿// This is an automatically generated file, based on settings.json and PackageSettingsGen.tt
```

Comment:
They should be regenerated automatically on build if they're missing or stale. Can probably kill them from source control, yeah...